<launch>
<arg name="seed" default="0"/>
<arg name="prob"  default="0.8"/>
<arg name="max_times"  default="5000"/>
<arg name="step"  default="0.02"/>
<arg name="index"  default="0"/>
<arg name="plan_times"  default="5"/>
<arg name="num"  default="1"/>

  <node pkg="dual_rrt_star" type="multi_plan_node" name="multi_plan_node" output="screen" required = "true" >
      <param name="dualrrt/bi_probility"       value="$(arg prob)" />
      <param name="dualrrt/max_sample_times"       value="$(arg max_times)" />
      <param name="dualrrt/task_step"       value="$(arg step)"/>
      <param name="dualrrt/constrain_index"       value="$(arg index)"/>
      <param name="dualrrt/seed"       value="$(arg seed)"/>
      <param name="dualrrt/planning_times" value="$(arg plan_times)"/>;
       <param name="dualrrt/outfile_name" value="$(arg num)"/>;
  </node>

</launch>

